### ISSUE TITLE
dependency orders

### ISSUE THREAD

**Issue by sfabriece** [mgechev/angular2-seed#1152]
Has anyone run into cases where the order of dependencies matters when bootstrapping?

I'm running into issues where I need to figure out the order in which different services are injected and making sure they are defined in that order when bootstrapping.

example
```
A->B-C
```

**Comment by mgechev** [comment]
Yes, the order of dependencies/imports matters. Take a look at this discussion https://github.com/angular/angular.io/issues/1301#issuecomment-233578238.
